README.md

```diff
@@ -239,6 +246,28 @@ Now where next? I would recommend my detailed tutorial where I spend time lookin
 
 Try it now: [Will it cluster? K3s on Raspbian](https://blog.alexellis.io/test-drive-k3s-on-raspberry-pi/)
 
+## Fetch your kubeconfig file from a k3s server
+
+```sh
+k3sup install --ip $IP --user user --skip-install
+```
+
+## Install and run k3s with HA using MySQL or PostgreSQL as external datasource
+
+If you want to have high availability of your k3s server you can use MySQL or PostgreSQL as the external datasource by pointing to your database instance with a few flags
+
+For MySQL as external data-store:
+```sh
+k3sup install --ip $IP -d sql --db-hostname $DBHOSTNAME --db-port $DBPORT --db-username $DBUSER --db-passwd $DBPASSWD --db-name $DBNAME
+```
+
+For PostgreSQL as external data-store:
```

Comment:
How do you join additional hosts with this approach? Have you tested SQL ha with more than one server?